tests/blivet_test.py

```diff
@@ -18,22 +18,44 @@ def test_bootloader_in_kickstart(self):
         in the kickstart data
         '''
 
+        # prepboot test case
         with patch('blivet.Blivet.bootLoaderDevice', new_callable=PropertyMock) as mockBootLoaderDevice:
             with patch('blivet.Blivet.mountpoints', new_callable=PropertyMock) as mockMountpoints:
                 # set up prepboot partition
                 bootloader_device_obj = PartitionDevice("test_partition_device")
                 bootloader_device_obj.size = Size('5MiB')
                 bootloader_device_obj.format = formats.getFormat("prepboot")
 
-                blivet_obj = Blivet()
+                prepboot_blivet_obj = Blivet()
 
                 # mountpoints must exist for updateKSData to run
                 mockBootLoaderDevice.return_value = bootloader_device_obj
                 mockMountpoints.values.return_value = []
 
                 # initialize ksdata
-                test_ksdata = returnClassForVersion()()
-                blivet_obj.ksdata = test_ksdata
-                blivet_obj.updateKSData()
+                prepboot_ksdata = returnClassForVersion()()
+                prepboot_blivet_obj.ksdata = prepboot_ksdata
+                prepboot_blivet_obj.updateKSData()
 
-        self.assertTrue("part prepboot" in str(blivet_obj.ksdata))
+        self.assertTrue("part prepboot" in str(prepboot_blivet_obj.ksdata))
```

Comment:
While modifying this line, change it to ``self.assertIn`` please. The same applies to a similar line below.